helpers/request.ts

```diff
@@ -149,11 +150,27 @@ export function corsRequest(url: string, mode: RequestMode = 'cors'): Request {
   return request;
 }
 
+function reload() {
+  window.location.reload();
+}
+
+function notifyWaitReload() {
+  const notification: Notification = {
+    canClose: false,
+    content: 'render::refresh-page',
+  };
+  getNotificationsCenter().add(notification);
+}
+
 function checkApplicationVersion(response: Response): boolean {
   const version = response.headers.get('Sonar-Version');
   if (version) {
     if (currentApplicationVersion && currentApplicationVersion !== version) {
-      window.location.reload();
+      if (getRequestOptions().refreshBehavior === RefreshBehavior.NotifyWaitReload) {
+        notifyWaitReload();
+      } else {
+        reload();
+      }
```

Comment:
I think that the whole part about the notification center is going a bit too far into putting pure SC stuff into SUC, and makes things more complex.
What about only adding the request options part in the init, and the option would just be a `versionReload` function that is called instead of doing `window.location.reload()` ? That request option would be completely optional, and the default if not provided would be to do like before.

I think it would simplify a bit and we would have less dependencies on SUC for the implem in SC, and it would still allow us to do whatever we want.